apps/builder/app/canvas/features/text-editor/text-editor.tsx

```diff
@@ -54,6 +55,56 @@ const AutofocusPlugin = () => {
   return null;
 };
 
+/**
+ * In case of text color is near transparent, make caret visible with color animation between #666 and #999
+ */
+const CaretColorPlugin = () => {
+  const [editor] = useLexicalComposerContext();
+  const caretClassName = useState(() => `a${nanoid()}`)[0];
```

Comment:
Why not destructuring?

Reply:
Copilot generated, not that bad, decided to leave